modules/network-observability-dns-overview.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_content-type: CONCEPT
+[id="network-observability-dns-overview_{context}"]
+= Configuring DNS tracking
+You can configure graphical representation of DNS tracking of network flows in the *Overview* view. Using DNS tracking with eBPF tracepoint hooks can serve various purposes:
+
+* Network Monitoring: Gain insights into DNS queries and responses, helping network administrators identify unusual patterns, potential bottlenecks, or performance issues.
+
+* Security Analysis: Detect suspicious DNS activities, such as domain name generation algorithms (DGA) used by malware, or identify unauthorized DNS resolutions that might indicate a security breach.
+
+* Troubleshooting: Debug DNS-related issues by tracing DNS resolution steps, tracking latency, and identifying misconfigurations.
+
+When DNS tracking is enabled, you can see the following metrics represented in a chart in the *Overview*. See the _Additional Resources_ of this section for more information about enabling and working with this view.
```

Comment:
```suggestion
When DNS tracking is enabled, you can see the following metrics represented in a chart in the *Overview*. See the _Additional Resources_  section for more information about enabling and working with this view.
```
...or another option is " See Additional Resources in this section for more information about...".